src/stratus-gwc/src/main/resources/geowebcache-rest-context.xml

```diff
@@ -2,23 +2,13 @@
 <beans xmlns="http://www.springframework.org/schema/beans"
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xmlns:context="http://www.springframework.org/schema/context"
-       xmlns:mvc="http://www.springframework.org/schema/mvc"
        xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
-    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-4.0.xsd
-    http://www.springframework.org/schema/mvc http://www.springframework.org/schema/mvc/spring-mvc-4.0.xsd">
+    http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-4.0.xsd">
 
   <!-- Used by org.geoserver.rest.RestConfiguration when setting up converters -->
-  <!-- DO NOT INCLUDE <mvc:annotation-driven> when updating this file - this conflicts with the Spring Boot context configuration
```

Comment:
This comment is kinda usefull when updating the file for future releases, maybe leave it in?

Reply:
It was a comment right in front of a bunch of also commented out `<mvc:annotation-driven>` elements.  I just removed them all, but I'll add the warning back in.